erpnext/shopping_cart/cart.py

```diff
@@ -83,20 +83,13 @@ def place_order():
 				if item.qty > item_stock.stock_qty[0][0]:
 					throw(_("Only {0} in Stock for item {1}").format(item_stock.stock_qty[0][0], item.item_code))
 
-	sales_order.flags.ignore_permissions = True
-	sales_order.insert()
-	sales_order.submit()
-
-	if hasattr(frappe.local, "cookie_manager"):
-		frappe.local.cookie_manager.delete_cookie("cart_count")
-
-	return sales_order.name
+	return quotation.name
 
 @frappe.whitelist()
 def request_for_quotation():
 	quotation = _get_cart_quotation()
 	quotation.flags.ignore_permissions = True
-	quotation.submit()
+	#quotation.submit()
```

Comment:
?

Reply:
commenting it out so that it doesn't get submitted when the function is called

Reply:
remove the line insted